src/zkevm_specs/evm_circuit/typing.py

```diff
@@ -779,23 +779,21 @@ def add(self, data: bytes, r: FQ) -> KeccakCircuit:
 
 class ExpCircuit:
     rows: List[ExpCircuitRow]
-    pad_rows: List[ExpCircuitRow]
+    max_exp_steps: int
+    OFFSET_INCREMENT = 7
```

Comment:
In the specs we don't really need the `OFFSET_INCREMENT`. That's used in the circuits to reduce the number of columns in the multiplication gadget. In the specs, we cover one step in a single row (unlike in the circuits). So this is not needed here.